cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+if [ -z "${KUBEVIRTCI_TAG}" ]; then
```

Comment:
I think we should check that either `KUBEVIRTCI_GOCLI_CONTAINER` or `KUBEVIRTCI_TAG` is provided.

Because below we have

```
${KUBEVIRTCI_GOCLI_CONTAINER:-kubevirtci/gocli:${KUBEVIRTCI_TAG}}
```

Reply:
Done, with an extra.